packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -156,20 +169,31 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     this[$onUpdate]();
   }
 
-  setAlphaCutoff(cutoff: number): void {
-    cutoff = Math.min(1.0, cutoff);
+  [$applyAlphaCutoff]() {
```

Comment:
@timmmeh I found some bugs as I was using these APIs in the editor. My fixes are here, so please give them a solid review.